src/components/LocationBias.tsx

```diff
@@ -11,17 +11,18 @@ import LoadingIndicator from '../icons/LoadingIndicator';
  * @public
  */
 export interface LocationBiasCssClasses {
-  container?: string,
+  locationBiasContainer?: string,
   location?: string,
   source?: string,
   button?: string,
   loadingIndicatorContainer?: string
 }
 
 const builtInCssClasses: Readonly<LocationBiasCssClasses> = {
-  container: 'text-sm text-neutral text-center flex justify-center items-center',
-  location: 'font-semibold mr-1',
-  button: 'text-primary hover:underline focus:underline ml-1',
+  locationBiasContainer: 'text-sm text-neutral text-center justify-center items-center flex flex-col lg:flex-row',
+  location: 'font-semibold mr-1 lg:ml-7',
```

Comment:
is this `mr-1` also for adding a space like `ml-1` was in `button` before? if so, maybe `whitespace-pre` would fix that too?